site/resources.html

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: Resources
+id: resources
+---
+<div class="sub-page">
+    <h3>Here you will find external resources about Contour, such as videos, podcasts, and community articles.</h3>
+    <div class="grid three">
+        <div class="col">
+            <div class="icon">
+                <img src="/img/Contour.svg" />
+            </div>
+            <div class="content">
+                <p class="strong">Demos and Deep Dives</p>
+                <p>We have a YouTube playlist for Contour demos and code deep dives, <a href="https://www.youtube.com/playlist?list=PL7bmigfV0EqRTmmjwWm4SxuCZwNvze7se">check it out here</a>.</p>
+            </div>
+        </div>
+        <div class="col">
+            <div class="embed-responsive embed-responsive-16by9">
+                <iframe class="embed-responsive-item" src="https://www.youtube.com/embed/jzla7xdqtwM"></iframe>
+              </div>
+            <div class="content">
+                <p><a href="https://www.youtube.com/embed/jzla7xdqtwM">Take Envoy Beyond a K8s Service Mesh - to Legacy Bare Metal & VMs + More</a></p>
```

Comment:
@amymanion ensure these link to the youtube URL, this is currently linking to the youtube `embed` url

Reply:
Fixed this.